File: app/companions/app/src/main/java/com/niutao/restaurant/activity/MenuDetailActivity.java
// MenuDetailActivity.kt
class MenuDetailActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu_detail)
        val name = findViewById<TextView>(R.id.name)
        val description = findViewById<TextView>(R.id.description)
        val price = findViewById<TextView>(R.id.price)
        val image = findViewById<ImageView>(R.id.image)
        val like = findViewById<Button>(R.id.like)
        val comment = findViewById<Button>(R.id.comment)

        like.setOnClickListener {
            // 点赞逻辑
        }

        comment.setOnClickListener {
            // 评论逻辑
        }
    }
}

File: app/companions/app/src/main/java/com/example/ntrestaurant/ui/MenuDetailActivity.kt
class MenuDetailActivity : AppCompatActivity() {
    private lateinit var menuRepository: MenuRepository

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu_detail)

        menuRepository = MenuRepositoryImpl()
        getMenu()
    }

    private fun getMenu() {
        val menu = menuRepository.getMenu()
        val menuId = intent.getStringExtra("menuId")
        val menuDetail = menu.find { it.id == menuId }
        if (menuDetail != null) {
            val description = menuDetail.description
            val price = menuDetail.price
            val image = menuDetail.image
            // 展示菜品详细信息
        }
    }
}
